R/print.R

```diff
@@ -0,0 +1,87 @@
+#' Print method for `<multi_epidist>` class
+#'
+#' @param x A `<multi_epidist>` object.
+#' @param n A `numeric` specifying how many `<epidist>` objects to print. This
+#' argument is passed to [head()] for `list` printing. Default is `NULL` and the
+#' number of elements to print is controlled by package [options()].
+#' @inheritParams print.epidist
+#'
+#' @return Invisibly returns a `<multi_epidist>`. Called for side-effects.
+#' @export
+print.multi_epidist <- function(x, ..., n = NULL) {
+  chkDots(...)
+
+  # determine how many <epidist> to print
+  if (is.null(n)) {
+    n <- getOption("epiparameter")$print_max
+    if (length(x) > n) {
+      n <- getOption("epiparameter")$print_min
+    } else {
+      n <- length(x)
+    }
+  }
+
+  extra_n <- length(x) - n
+  n_entries <- length(x) # nolint object_usage_linter used by cli {} syntax
+
+  diseases <- vapply(x, "[[", FUN.VALUE = character(1), "disease")
+  alpha_unique_diseases <- sort(unique(diseases))
```

Comment:
Default `sort()` is locale-dependent. Is it what we want? If not, `sort(method = "radix")` is a better, locale-independent alternative.

Reply:
Here I am using `sort()` to alphabetise the vector of characters. I think locale-dependent sorting would be best for alphabetical order across different languages, even if that differs between locales, but please correct me if I've misunderstood this.